rskj-core/src/test/java/co/rsk/rpc/modules/eth/EthModuleTest.java

```diff
@@ -75,7 +112,7 @@ public void callSmokeTest() {
                         null, null, null));
 
         String result = eth.call(args, "latest");
-        assertThat(result, is(TypeConverter.toJsonHex(hreturn)));
+        assertThat(result, is(TypeConverter.toUnformattedJsonHex(hReturn)));
```

Comment:
I would not test agains the function, but for the explicit value

Reply:
done